app/components/linkable-header.hbs

```diff
@@ -0,0 +1,6 @@
+<h2 id={{@linkTarget}} class="heading heading--linkable" tabindex="0">
```

Comment:
Hmm, we may need to support other heading levels like h3's and such https://emberjs-1.gitbook.io/ember-component-patterns/general-considerations/considering-forms

Reply:
Yes, I agree. I'd love some ideas on the most efficient way to achieve that in the Ember paradigm.

Reply:
Hmm... the more I think about it (and ask about it in the Ember Discord) the more I realize this is a hole in Ember/Glimmer...

For now, I think it's okay to leave it as a `h2` and we can write a follow-up issue where this is discussed in finer detail to not hold up this PR.